src/main/java/com/m3/curly/Request.java

```diff
@@ -224,20 +224,22 @@ public String getContentType() {
         return getRequestBody().getContentType();
     }
 
+    @SuppressWarnings("unchecked")
     public Map<String, Object> getFormParams() {
-        return formParams;
+        return (Map<String, Object>) formParams;
     }
```

Comment:
By （good） rights, We should change this return type to `Map<String, ?>`

However I kept the compatibility.